src/AutoRest.CSharp/MgmtTest/Generation/MgmtBaseTestWriter.cs

```diff
@@ -556,7 +538,7 @@ public void WriteExampleValue(CodeWriter writer, CSharpType cst, ExampleValue ex
             }
         }
 
-        public static string useVariableName(string variableName)
+        public string useVariableName(string variableName)
         {
             if (Utilities.StringExtensions.IsCSharpKeyword(variableName))
```

Comment:
Looks like we are trying to avoid multi-defined variables here. Could you please try to use `CodeWriterDeclaration`? In the `CodeWriter` class when it is writing code, it will do the deduplication for your variables.
Therefore I believe we could remove this function, and from where we are invoking this, we just write `var something = new CodeWriterDeclaration("variableName");`, and writes it using the format symbol `:D`, like
```
writer.Line($"var {something:D} = Id");
```
and code writer could take care of the local variable names in the same scope.
```